gammapy/spectrum/core.py

```diff
@@ -374,6 +386,35 @@ def excess_vector(self):
         """
         return self.on_vector + self.off_vector * self.alpha * -1
 
+    def peek(self, figsize=(15, 5)):
+        """Quick-look summary plots."""
+        import matplotlib.pyplot as plt
+        plt.style.use('ggplot') 
+
+        fig, ((ax1, ax2), (ax3, ax4)) = plt.subplots(2,2,figsize=figsize)
+        self.on_vector.plot(ax=ax1, label='Total counts', histtype='step',
+                            color='darkred', alpha=0.8, lw=2)
+        self.off_vector.plot(ax=ax1, label='Background estimate',
+                             histtype='step', color='darkblue',
+                             alpha=0.8, lw=2)
+        ax1.legend(numpoints=1)
+        ax1.set_title('Counts')
+        # TODO: Replace with total spectrum stats
+        ax2.text(0.2, 0.9, 'Exposure: {}'.format(self.exposure))
+        ax2.text(0.2, 0.85, 'Alpha: {}'.format(self.alpha))
+        ax2.axis('off')
+        ax2.set_title('Spectrum Stats')
+        self.aeff.plot(ax=ax3)
+        ax3.set_title('Effective Area')
+        self.edisp.plot_matrix(ax=ax4)
+        ax4.set_title('Energy Dispersion')
+
+        import seaborn as sns
```

Comment:
Please remove the `seaborn` dependency for now.

We can discuss if we want to add it, I'm not really against this. But if we do, let's use a separate PR and do it right (i.e. declare the dependency in `setup.py`, document it in the install notes, add it to travis-ci builds).